### PR TITLE
Include with notice about where versioned docs are for SignalR

### PR DESCRIPTION
**Pull request by rachelappel**
Fixes https://github.com/aspnet/Docs/issues/5682

### CODE REVIEW

**Comment by rachelappel** [comment]
@Rick-Anderson  This is just the include. I'll open a PR to actually include it in files once this is merged. That way it will avoid merge issues.

File: aspnetcore/includes/signalr-version-notice.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+> [!NOTE]
+> This document details the latest version of ASP.NET Core SignalR. Looking to get started with an earlier version of SignalR? See the [SignalR 1.x documentation](xref:aspnet/signalr/).
```
Comment (Rick-Anderson) [inline aspnetcore/includes/signalr-version-notice.md]: The PU prefers the more ref style.

This document details the latest version of ASP.NET Core SignalR. ~Looking to get started with an earlier version of SignalR?~ See the [SignalR 1.x documentation](xref:aspnet/signalr/) for the ASP.NET Framework version.

Comment (rachelappel) [reply aspnetcore/includes/signalr-version-notice.md]: Ok. I took it from here https://docs.microsoft.com/en-us/aspnet/core/getting-started
I'll add an issue to change it there too then.

**Comment by Rick-Anderson** [comment]
I reopened #5682 until you add the include. Make sure it goes in both topics.

File: aspnetcore/includes/signalr-version-notice.md [hunk 2]
```diff
@@ -0,0 +1,2 @@
+> [!NOTE]
+> This document details the latest version of ASP.NET Core SignalR. See the [SignalR 1.x documentation](../../aspnet/signalr/) for the ASP.NET Framework version.
```
Comment (Rick-Anderson) [inline aspnetcore/includes/signalr-version-notice.md]: Don't use relative links. I think the (aspnet/aspnet/signalr) is what you want. @guardrex  is that right?

Comment (guardrex) [reply aspnetcore/includes/signalr-version-notice.md]: Yes ... that will land here :point_right: https://docs.microsoft.com/en-us/aspnet/signalr/

**Pull request merged**